### PR TITLE
CLN: remove merge-pr script, clean Exception in tm, doc

### PR DESCRIPTION
**Pull request by jbrockmendel**
I count 131 of these ATM, some of which we'll end up marking as intentionally non-specific.

### CODE REVIEW

File: scripts/merge-pr.py [hunk 1]
```diff
@@ -310,12 +310,6 @@ def fix_version_from_branch(branch, versions):
 
 print("\n=== Pull Request #%s ===" % pr_num)
 
-# we may have un-printable unicode in our title
-try:
-    title = title.encode("raw_unicode_escape")
-except Exception:
-    pass
-
```
Comment (gfyoung) [inline scripts/merge-pr.py]: @pandas-dev/pandas-core : Out of curiosity, how many people actually use this nowadays?

**Comment by jorisvandenbossche** [comment]
I never use it anymore, and I think we fully switched to using github's button to merge PRs. But maybe some people still use it to fetch PR's branchs and push back there?

File: scripts/merge-pr.py (hunk 1 above)
Comment (datapythonista) [reply scripts/merge-pr.py]: Never used it, happy to get rid of it if nobody does.

**Comment by jreback** [comment]
yeah i think we can blow away the merge script

**Comment by jbrockmendel** [comment]
Updated to remove merge-pr.py

File: pandas/tests/groupby/aggregate/test_other.py [hunk 2]
```diff
@@ -385,10 +385,7 @@ def test_agg_consistency():
     # agg with ([]) and () not consistent
     # GH 6715
     def P1(a):
-        try:
-            return np.percentile(a.dropna(), q=1)
-        except Exception:
-            return np.nan
+        return np.percentile(a.dropna(), q=1)
```
Comment (WillAyd) [inline pandas/tests/groupby/aggregate/test_other.py]: Can you just get rid of this function altogether now?

File: pandas/tests/test_downstream.py [hunk 3]
```diff
@@ -50,6 +50,7 @@ def test_dask(df):
     assert ddf.compute() is not None
 
 
+@pytest.mark.filterwarnings("ignore:Panel class is removed")
```
Comment (WillAyd) [inline pandas/tests/test_downstream.py]: Was this not causing CI failures before? Thought we failed on warnings like this

**Review by WillAyd — CHANGES_REQUESTED** [review]
Looks good - minor comments

File: pandas/tests/groupby/aggregate/test_other.py (hunk 2 above)
Comment (jbrockmendel) [reply pandas/tests/groupby/aggregate/test_other.py]: This gets passed to `agg` twice.  I think its marginally clearer like this than writing the lambda twice.

File: pandas/tests/test_downstream.py (hunk 3 above)
Comment (jbrockmendel) [reply pandas/tests/test_downstream.py]: this is showing up locally

File: pandas/tests/groupby/aggregate/test_other.py (hunk 2 above)
Comment (WillAyd) [reply pandas/tests/groupby/aggregate/test_other.py]: Ah OK didn't see in diff but makes sense

**Review by WillAyd — APPROVED** [review]
lgtm

File: pandas/tests/test_downstream.py (hunk 3 above)
Comment (jorisvandenbossche) [reply pandas/tests/test_downstream.py]: This should not be needed with latest xarray I suppose

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
Thanks!

**Pull request merged**